tenable/io/v3/vm/networks.py

```diff
@@ -0,0 +1,329 @@
+'''
+Networks
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`networks <networks>` API endpoints.
+
+Methods available on ``tio.v3.vm.networks``:
+
+.. rst-class:: hide-signature
+.. autoclass:: NetworksAPI
+    :members:
+'''
+from typing import Dict, List, Union
+
+from tenable.errors import UnexpectedValueError
+from tenable.io.base import TIOEndpoint, TIOIterator
+
+
+class NetworksIterator(TIOIterator):
+    '''
+    The networks iterator provides a scalable way to work through networks
+    result sets of any size.  The iterator will walk through each page
+    of data, returning one record at a time.  If it reaches the end of a
+    page of records, then it will request the next page of information and
+    then continue to return records from the next page (and the next, and the
+    next) until the counter reaches the total number of records that the API
+    has reported.
+
+    Attributes:
+        count (int): The current number of records that have been returned
+        page (list):
+            The current page of data being walked through.  pages will be
+            cycled through as the iterator requests more information from the
+            API.
+        page_count (int): The number of record returned from the current page.
+        total (int):
+            The total number of records that exist for the current request.
+    '''
+    pass
+
+
+class NetworksAPI(TIOEndpoint):
+    '''
```

Comment:
@harshshah-tenable we have to use the latest explore-base endpoint and for iterator keep it notImplemented.

Reply:
Updated